ui/app/[locale]/(authorized)/kafka/[kafkaId]/overview/page.tsx

```diff
@@ -25,6 +25,7 @@ export default function OverviewPage({ params }: { params: KafkaParams }) {
     "outgoingByteRate",
     "incomingByteRate",
   ]);
+  const topics = getTopics(params.kafkaId, { fields: "status", pageSize: 1 });
```

Comment:
Only select the minimum number of records because we only care about the summary meta data for this request, not the list of topics.